pkg/cmd/server/start/start_master.go

```diff
@@ -464,7 +464,9 @@ func (m *Master) Start() error {
 			openshiftConfig.AppInformers.Start(utilwait.NeverStop)
 			openshiftConfig.AuthorizationInformers.Start(utilwait.NeverStop)
 			openshiftConfig.ImageInformers.Start(utilwait.NeverStop)
-			openshiftConfig.TemplateInformers.Start(utilwait.NeverStop)
+			if openshiftConfig.Options.TemplateServiceBrokerConfig != nil {
```

Comment:
This isn't right - you should be avoiding the call to `TemplateInformers.Template()`.  Start is already guarded by "has anyone asked for this informer yet".